src/lib/schematics/update/material/data/method-call-checks.ts

```diff
@@ -18,38 +18,6 @@ export interface MaterialMethodCallData {
 }
 
 export const methodCallChecks = transformChanges<MaterialMethodCallData>([
-  {
-    pr: 'https://github.com/angular/material2/pull/9190',
-    changes: [
-      {
-        className: 'NativeDateAdapter',
-        method: 'constructor',
-        invalidArgCounts: [
-          {
-            count: 1,
-            message: '"g{{platform}}" is now required as a second argument'
-          }
-        ]
-      }
-    ]
-  },
-
-  {
-    pr: 'https://github.com/angular/material2/pull/10319',
-    changes: [
-      {
-        className: 'MatAutocomplete',
-        method: 'constructor',
-        invalidArgCounts: [
-          {
-            count: 2,
-            message: '"g{{defaults}}" is now required as a third argument'
```

Comment:
FYI: There will be a follow-up that add some additional messages to the TSLint failure. Even though it's not necessarily needed, it's good to explain people what changed.